packages/angular-workspace/nimble-angular/src/thirdparty/directives/checkbox_value_accessor.ts

```diff
@@ -14,10 +14,10 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Directive} from '@angular/core';
+import {Directive, forwardRef, Provider} from '@angular/core';
```

Comment:
Just double checking my understanding on where this diff is coming from since it looks odd on the surface (adding new imports that aren't used).

1. At some point in the past we commented out the code below that uses these imports.
1. Then either the linter/prettier told us to remove unused imports or someone did it manually. (Same for the spacing changes below)
2. Now you copied the latest source from the Angular repo and re-commented out the unused code
3. But we're not bothering to re-apply the import deletions and spacing changes since the linter currently ignores this code

If that's accurate, I think these diffs make sense and are a good step since they align us more closely with the Angular source and so will reduce future diffs in future upgrades.

So no changes necessary if that's accurate but please help educate me if there's some other process involved.

Reply:
That's exactly right. I previously had removed any unused imports (possibly just out of habit), but now I figure it's better to leave them as long as they don't cause any problems.